cmake/Hunter/config.cmake

```
@@ -1,2 +1,7 @@
 hunter_config(CURL VERSION ${HUNTER_CURL_VERSION} CMAKE_ARGS HTTP_ONLY=ON CMAKE_USE_OPENSSL=OFF CMAKE_USE_LIBSSH2=OFF CURL_CA_PATH=none)
-hunter_config(Boost VERSION 1.66.0)
+hunter_config(
+    Boost
+    VERSION 1.66.0_new_url
+    SHA1 f0b20d2d9f64041e8e7450600de0267244649766
+    URL https://boostorg.jfrog.io/artifactory/main/release/1.66.0/source/boost_1_66_0.tar.gz
+)
```

A better way of doing this would be to upgrading hunter to the latest version, by changing lines https://github.com/ethereum-mining/ethminer/blob/ce52c74021b6fbaaddea3c3c52f64f24e39ea3e9/CMakeLists.txt#L16-L20 to:

```cmake
HunterGate(
    URL "https://github.com/cpp-pm/hunter/archive/v0.23.320.tar.gz"
    SHA1 "9b4e732afd22f40482c11ad6342f7d336634226f"
    LOCAL
)
```

and specifying a version for `jsoncpp` (the latest version by hunter changes linking).

```suggestion
hunter_config(Boost VERSION 1.66.0)
hunter_config(jsoncpp VERSION 1.8.0)
```

But this also works!